view/frontend/web/js/utils/cart-utils.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+define(['Magento_Customer/js/customer-data'], function (customerData) {
+  'use strict'
+
+  const getCartItems = function () {
+    return customerData.get('cart')().items
+  }
+
+  const refreshMiniCart = function () {
+    const sectionsToUpdate = ['cart']
+    customerData.invalidate(sectionsToUpdate)
+    customerData.reload(sectionsToUpdate, true)
+  }
+
+  const mapToExtendCartItem = function (magentoCartItem) {
+    return {
+      name: magentoCartItem.product_name,
+      sku: magentoCartItem.product_sku,
+      qty: magentoCartItem.qty,
+      price: magentoCartItem.product_price_value * 100,
+      item_id: magentoCartItem.product_id,
+      options: [],
```

Comment:
The `options` property of the Magento cart item was always an empty array while debugging so I couldn't figure out its shape to map to the Extend cart item that the add on expects. I'll try to track it down but leaving it empty didn't seem to cause any problems